src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -196,10 +196,16 @@ private static DependencyGraphSpec ToDependencyGraphSpec(ProjectNames projectNam
                     .Any(tfm => tfm.Version >= Version20);
                 var toolFramework = isNetCore20 ? CommonFrameworks.NetCoreApp20 : CommonFrameworks.NetCoreApp10;
 
+                var packagesPath = GetRestoreProjectPath(projectRestoreInfo.TargetFrameworks);
+                var fallbackFolders = GetRestoreFallbackFolders(projectRestoreInfo.TargetFrameworks).ToList();
```

Comment:
It's valid here, but for some of the others(sources) it makes more sense to keep it as an array. The utility methods are arrays, so we'd end up doing array -> list -> enum -> list.